### ISSUE TITLE
[Broken] Youtube 

### ISSUE THREAD

**Issue by W4lv1s** [blackjack4494/yt-dlc#50]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dlc version **2020.10.26**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
 youtube-dlc -x --audio-format mp3 https://www.youtube.com/watch?v=0Qd_k0hv3Nw -v
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-x', '--audio-format', 'mp3', 'https://www.youtube.com/watch?v=0Qd_k0hv3Nw', '-v']
[debug] Loading archive file None
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dlc version 2020.10.26
[debug] Python version 3.9.0 (CPython) - macOS-10.13.6-x86_64-i386-64bit
[debug] exe versions: avconv 12.3, avprobe 12.3, ffmpeg present, ffprobe present, rtmpdump Reason
[debug] Proxy map: {}
[youtube] 0Qd_k0hv3Nw: Downloading webpage
[youtube] 0Qd_k0hv3Nw: Downloading embed webpage
ERROR: Unable to extract JS player URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dlc with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/youtube_dlc/YoutubeDL.py", line 830, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/lib/python3.9/site-packages/youtube_dlc/extractor/common.py", line 532, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/lib/python3.9/site-packages/youtube_dlc/extractor/youtube.py", line 2065, in _real_extract
    jsplayer_url_json = self._search_regex(
  File "/usr/local/lib/python3.9/site-packages/youtube_dlc/extractor/common.py", line 1010, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dlc.utils.RegexNotFoundError: Unable to extract JS player URL; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dlc with the --verbose flag and include its complete output.
```

## Description

As of yesterday downloading anything from Youtube seems to fail.

**Comment by blackjack4494** [comment]
search before posting.

**Comment by nixxo** [comment]
fixed in #35
